scripts/triggers-end-to-end-tests/triggers/index.js

```diff
@@ -70,6 +95,18 @@ exports.writeToAuth = functions.https.onRequest(async (req, res) => {
   res.json({ created: "ok" });
 });
 
+exports.createUserFromAuth = functions.https.onRequest(async (req, res) => {
+  await createUserWithEmailAndPassword(auth, "email@gmail.com", "mypassword");
```

Comment:
tangent - I think you can create user using the Admin SDK (https://firebase.google.com/docs/auth/admin/manage-users#create_a_user)

Reply:
Unfortunately, using the Admin SDK here will not trigger auth blocking functions

Reply:
Dang... does that make sense? Is that how it works in prod as well?